Client/Assets/About/Eula.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
```

Comment:
Is there a reason that this file is not under the Cliqz resources?

Reply:
Moved to `Cliqz/Resources`